test_utils/basetest.py

```diff
@@ -46,38 +20,62 @@ def resource_group_exists(self, group_name):
         return False
 
     def delete_resource_group(self):
-        resource_group_params = {'location': self.location}
-        resp = self.resource_client.resource_groups.delete(
-            self.RESOURCE_GROUP_NAME, resource_group_params)
-        resp.wait()
-        print('Deleting {}'.format(self.RESOURCE_GROUP_NAME), resp.status())
+        try:
+            resp = self.resource_client.resource_groups.begin_delete(self.RESOURCE_GROUP_NAME)
+            resp.wait()
+            print('Deleting {}'.format(self.RESOURCE_GROUP_NAME), resp.status())
```

Comment:
after wait we expect the resource group to be deleted completely right? If yes then change Deleting ->  Deleted

Reply:
also do not wrap it in try catch we want the error to propagate to the top so that the test exits

Reply:
updated